plugin/web/extensions/location/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+import { PostMessageProps } from "../types";
+
+export function postMsg({ action, payload }: PostMessageProps) {
+  parent.postMessage(
+    {
+      action,
+      payload,
+    },
+    "*",
+  );
+}
```

Comment:
Let's move utils to the global scope (not based on extension). Since a lot of them will be used in most extensions like postMsg.
Can you make a quick PR moving Sidebar utils in to `web/` so you can use postMsg, etc in this PR?

Reply:
pr : https://github.com/eukarya-inc/reearth-plateauview/pull/44

Reply:
agreed to keep it now as it is